plugin/src/test/java/io/jenkins/plugins/analysis/warnings/recorder/AffectedFilesResolverITest.java

```diff
@@ -232,55 +149,35 @@ public void shouldShowFileOutsideWorkspaceIfConfigured() {
         recorder.setSourceDirectory(buildsFolder);
 
         // First build: copying the affected file is forbidden
-        TableRowPageObject unresolvedRow = buildAndVerifyFilesResolving(job,
-                "0 copied", "1 not in workspace", "0 not-found", "0 with I/O error");
+        buildAndVerifyFilesResolving(job,"0 copied", "1 not in workspace", "0 not-found", "0 with I/O error");
 
         AnalysisResult result = getAnalysisResult(job.getLastCompletedBuild());
         assertThat(result.getErrorMessages()).contains(
                 String.format("Additional source directory '%s' must be registered in Jenkins system configuration",
                         buildsFolder));
 
-        assertThat(unresolvedRow.hasLink(AFFECTED_FILE)).isFalse();
 
         WarningsPluginConfiguration.getInstance().setSourceDirectories(
                 Collections.singletonList(new SourceDirectory(buildsFolder)));
 
         // Second build: copying the affected file is permitted
-        TableRowPageObject resolvedRow = buildAndVerifyFilesResolving(job,
-                "1 copied", "0 not in workspace", "0 not-found", "0 with I/O error");
-
-        assertThat(resolvedRow.hasLink(AFFECTED_FILE)).isTrue();
-
-        SourceCodeView sourceCodeView = openSourceCode(resolvedRow);
-        assertThat(sourceCodeView.getSourceCode()).contains("<io.jenkins.plugins.analysis.core.steps.IssuesRecorder>");
+        buildAndVerifyFilesResolving(job,"1 copied", "0 not in workspace", "0 not-found", "0 with I/O error");
     }
 
-    private SourceCodeView openSourceCode(final TableRowPageObject resolvedRow) {
-        return new SourceCodeView(resolvedRow.clickColumnLink(AFFECTED_FILE));
-    }
-
-    private TableRowPageObject buildAndVerifyFilesResolving(final FreeStyleProject job, final String... resolveMessages) {
+    private void buildAndVerifyFilesResolving(final FreeStyleProject job,
+            final String... resolveMessages) {
         AnalysisResult result = scheduleBuildAndAssertStatus(job, Result.SUCCESS);
 
         assertThat(getConsoleLog(result)).contains(resolveMessages);
 
-        TablePageObject issues = getIssuesTable(result);
-        assertThat(issues.getColumnHeaders()).containsExactly(
-                DETAILS, AFFECTED_FILE, SEVERITY, AGE);
-        List<TableRowPageObject> rows = issues.getRows();
-        assertThat(rows).hasSize(1);
-        TableRowPageObject row = rows.get(0);
-        assertThat(row.getValuesByColumnLabel()).contains(
-                entry(DETAILS, "foo defined but not used"),
-                entry(AFFECTED_FILE, "config.xml:451"),
-                entry(SEVERITY, "Normal"));
-        return row;
-    }
+        assertThat(result.getIssues()).hasSize(1);
```

Comment:
Da wäre noch die Idee gewesen, die Table zu prüfen, analog zu meinem Kommentar  in https://github.com/jenkinsci/warnings-ng-plugin/pull/466#discussion_r426424677

Reply:
Entsprechenden Test auf Severity und Filename hinzugefügt.